src/reducers/computeRenderedColumns.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright Schrodinger, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule computeRenderedColumns
+ */
+
+'use strict';
+
+import clamp from 'lodash/clamp';
+import columnWidths from 'columnWidths';
+import scrollbarsVisibleSelector from 'scrollbarsVisible';
+import { updateColumnWidth, updateColumnGroupWidth } from 'updateColumnWidth';
+
+export default function computeRenderedColumns(state, columnAnchor) {
```

Comment:
similar to [computeRenderedRows](https://github.com/schrodinger/fixed-data-table-2/blob/v1.0-beta/src/reducers/computeRenderedRows.js), but more cleaned up